rules/windows/process_creation/proc_creation_win_susp_powercfg.yml

```
@@ -0,0 +1,34 @@
+title: Suspicious Powercfg Execution 
```

If you can do other things with `powercfg` I think its better to add this to the title to be more clear and in case we add other susp rule for `powercfg`

```suggestion
title: Suspicious Powercfg Execution To Change Lock Screen Timeout
```

If you accept this change. Also change the filename by adding the `lock_screen` to the end